### DISCUSSION TITLE
telegram received but incomplete or with errors, since rtl_wmbus reports that CRC checks failed

### DISCUSSION THREAD

**Discussion by geogeozib57** [wmbusmeters/wmbusmeters#939] [Q&A]
Good morning ,

I installed wmbuslmeters on a raspberry pi 3 b+, the whole installation went well but when I try to capture my water meter I have the following message:
 "telegram received but incomplete or with errors, since rtl_wmbus reports that CRC checks failed"
, my meter is of the "DIEHL" brand and the transmission frequency is 169 Mhz.
Thank you in advance for your help.

Raspberry pi 3b+
FM TV Dvb-T Stick RTL2832U

![image](https://user-images.githubusercontent.com/89994079/230651345-12416257-7bf5-4468-862e-a93ba5b5172a.png)
![IMG_20230323_131720](https://user-images.githubusercontent.com/89994079/230651525-70f6efb9-f783-45f9-bd3e-7ab2d2177c54.jpg)
![IMG_20230323_122213](https://user-images.githubusercontent.com/89994079/230651616-d53ea07e-4cc1-48cf-b1f1-a196417e8c36.jpg)

**Comment by geogeozib57** [comment]
no one has any ideas?

**Reply by BIBOLV** [reply]
You can search issue archive - this topic has been discussed at least few times. [Here](https://wmbusmeters.github.io/wmbusmeters-wiki/) are few topics about antennas.
So basically try moving closer to meter and check if CRC issues disappear. That is usually due to lost message frames during reception.

**Reply by geogeozib57** [reply]
thank you for your answer, I have already tried to bring the antenna closer but the result is the same.

**Comment by weetmuts** [comment]
It might also be that the meter transmits on 169MHz and on this frequency the modulation is called n-mode.
rtl-wmbus does not support n-mode telegrams. The dongle amb3665-M does though.

**Reply by geogeozib57** [reply]
yes indeed it is on the 169mhz frequency, so the only way is to have an amb3665-m dongle or with those it is possible that it works?
